native-image-tests/build.gradle

```diff
@@ -20,7 +20,7 @@ dependencies {
   implementation project(':mockwebserver-deprecated')
   implementation project(':okhttp-urlconnection')
   implementation project(':mockwebserver-junit4')
-  implementation project(':mockwebserver-junit5')
+  testRuntimeOnly project(':mockwebserver-junit5')
```

Comment:
I don't think this is right, native image tests is not generally running at test configuration time.  It isn't part of the intellij setup (see settings.gradle) because Intellij doesn't understand that source files can be in two modules at once.  I'll try to find a better fix for this.